packages/manager/apps/pci-ssh-keys/src/pages/ListingPage.tsx

```diff
@@ -145,6 +150,13 @@ export default function ListingPage() {
           variant={ODS_BUTTON_VARIANT.stroked}
           color={ODS_THEME_COLOR_INTENT.primary}
           href={hrefAdd}
+          onClick={() => {
+            trackClick({
+              name: 'PCI_PROJECTS_SSH_KEYS_ADD',
+              type: 'action',
+              level2: PCI_LEVEL2,
```

Comment:
there is no possibility to set level2 by default depending of the uapp ? because level2 will be always the same on this app

Reply:
for this µapp it'll be fine since there is basically just a single trackClick, i agree that it should be possible to set up level2 globally, we need to dig this for future apps